prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -76,20 +76,47 @@ object ConstantSubstitutor {
  * Custom resolver for the FHIR path engine.
  */
 class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext {
-    override fun resolveConstant(appContext: Any?, name: String?, beforeContext: Boolean): Base {
+    override fun resolveConstant(appContext: Any?, name: String?, beforeContext: Boolean): Base? {
         // Name is always passed in from the FHIR path engine
         require(!name.isNullOrBlank())
 
         return when {
             appContext == null || appContext !is CustomContext ->
                 throw PathEngineException("No context available to resolve constant '$name'")
+
+            // Support prefix constant replacement like for URLs similar to %ext in FHIRPathEngine
```

Comment:
Might be worth adding the example of using this for extensions. Something like:
```// for example: resource: '%resource.source.extension(%`rsext-source-software-vendor-org`).value' ```

Reply:
Will add.